test/e2e/specs/fakerp/etcdrecovery.go

```diff
@@ -101,7 +93,7 @@ var _ = Describe("Etcd Recovery E2E tests [EtcdRecovery][Fake][LongRunning]", fu
 						Containers: []v1.Container{
 							{
 								Name:            "etcdbackup",
-								Image:           cs.Config.Images.EtcdBackup,
+								Image:           "quay.io/openshift-on-azure/etcdbackup:latest",
```

Comment:
We will not be able to test PR changes to the backup process with this change. Should likely use an env variable and default to this spec if not set.